### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by Dmitry-Erokhin**
# One-line summary
Fixing typo

### CODE REVIEW

**Comment by dstockhammer** [comment]
:+1:

File: docs/README.md [hunk 1]
```diff
@@ -209,8 +209,8 @@ Where:
 - `id` - just a name, string with no additional constraints
```
Comment (dstockhammer) [inline docs/README.md:209]: This list item is inconsistent, since it has no period at the end. Not sure how strict you want to be here, but at least I've pointed it out :)

File: docs/README.md [hunk 2]
```diff
@@ -209,8 +209,8 @@ Where:
...
 - `target` - the environment this step is executed against. The environment must
   be present in the [configuration](#configuration) file.
-- `paths` - array of paths to be processed. Each path will be scanned (non
-  recursively) for yaml files (`*.yaml` or `*.yml`). Each found yaml file is
+- `paths` - an array of paths to be processed. Each path will be scanned (non
+  recursively) for YAML files (`*.yaml` or `*.yml`). Each found YAML file is
   considered to be a resource manifest.
 - `env` - object with key-value pairs used for processing template variables in
```
Comment (dstockhammer) [inline docs/README.md:215]: should be `an object` for consistency

File: docs/README.md [hunk 3]
```diff
@@ -160,11 +160,11 @@ process:
 #...
 ```
 
-If a variable, found in manifest is not defined in the context – the process
+If a variable, found in the manifest is not defined in the context – the process
```
Comment (dstockhammer) [inline docs/README.md:163]: no comma?

File: docs/README.md [hunk 4]
```diff
@@ -160,11 +160,11 @@ process:
...
 will be interrupted.
 
 ### Includes
-Manifest can include other yaml file
+Manifest can include other YAML file
```
Comment (dstockhammer) [inline docs/README.md:167]: s/file/files

Comment (dstockhammer) [reply docs/README.md:167]: `The` manifest?

File: docs/README.md [hunk 5]
```diff
@@ -110,7 +110,7 @@ or subscription.
 
 In case of batch processing (see [Batch processing](#batch-processing)), context
```
Comment (dstockhammer) [inline docs/README.md:111]: `the` context?

File: docs/README.md [hunk 6]
```diff
@@ -62,10 +62,10 @@ Clin supports multiple environments that have to be configured and specified on
 every command.
 
 ## Configuration
-Clin uses configuration using a yaml file located either in the command's
+Clin uses configuration using a YAML file located either in the command's
```
Comment (dstockhammer) [inline docs/README.md:65]: Clin uses configuration ~using~ `from` a YAML file...

or

Clin is configured using a Yaml file..

File: docs/README.md [hunk 7]
```diff
@@ -11,24 +11,24 @@
...
 `{{TEMPLATE_VARIABLES}}` and `@@@./includes.yaml` , optionally enforced by
-[yaml anchors](https://www.google.com/search?q=yaml+anchors) enable expressive
+[YAML anchors](https://www.google.com/search?q=yaml+anchors) enable expressive
 and DRY definition of the Nakadi infrastructure.
 
 For more details see [Manifests format](#manifests-format).
 
 ### Dry-run by default
-Each command that can possibly change Nakadi resources in NOT performing any
+Each command that may change Nakadi resources is NOT performing any
```
Comment (dstockhammer) [inline docs/README.md:31]: I would rephrase this to something like:

Commands do NOT execute any changes (write operations?) by default.

File: docs/README.md [hunk 8]
```diff
@@ -11,24 +11,24 @@
 - [Dumping](#dumping)
 
 ## Core concepts
-**clin** sends http requests to Nakadi to create or update resources. The source
-of data for this is yaml manifest file which corresponds to a single Nakadi
+**clin** sends HTTP requests to Nakadi to create or update resources. The source
+of data for this is YAML manifest file which corresponds to a single Nakadi
```
Comment (dstockhammer) [inline docs/README.md:15]: `a` or `the` YAML manifest?

File: docs/README.md [hunk 9]
```diff
@@ -90,10 +90,10 @@ Kind defines the type of Nakadi resource. Currently supported:
 properties.
 
 ### Template variables
-Manifest yaml can include template variables in double curly braces (`{{VAR}}`)
+Manifest YAML can include template variables in double curly braces (`{{VAR}}`)
```
Comment (dstockhammer) [inline docs/README.md:93]: `The` or `a` manifest

File: docs/README.md (hunk 2 above)
Comment (Dmitry-Erokhin) [reply docs/README.md:215]: 👍

File: docs/README.md (hunk 3 above)
Comment (Dmitry-Erokhin) [reply docs/README.md:163]: Probably... Will remove.

File: docs/README.md (hunk 4 above)
Comment (Dmitry-Erokhin) [reply docs/README.md:167]: 👍

File: docs/README.md (hunk 7 above)
Comment (Dmitry-Erokhin) [reply docs/README.md:31]: Noted, will rephrase. Thanks.

File: docs/README.md [hunk 10]
```diff
@@ -90,10 +89,10 @@ Kind defines the type of Nakadi resource. Currently supported:
 properties.
 
 ### Template variables
-Manifest yaml can include template variables in double curly braces (`{{VAR}}`)
+Then manifest YAML can include template variables in double curly braces (`{{VAR}}`)
```
Comment (dstockhammer) [inline docs/README.md:92]: `Then`?

File: docs/README.md [hunk 11]
```diff
@@ -11,26 +11,25 @@
...
 and DRY definition of the Nakadi infrastructure.
 
 For more details see [Manifests format](#manifests-format).
 
 ### Dry-run by default
-Each command that can possibly change Nakadi resources in NOT performing any
-changes by default. To apply actual changes, additional key `-X` or `--execute`
-should be specified:
+Commands do NOT apply any changes by default. To apply changes, 
+additional key `-X` or `--execute` should be specified:
```
Comment (dstockhammer) [inline docs/README.md:32]: `the` additional key, probably also `the` changes. Though maybe consider rephrasing to something like this:

To apply the changes, pass the `-X` or `--execute` flag to the command:

File: docs/README.md [hunk 12]
```diff
@@ -62,10 +61,10 @@ Clin supports multiple environments that have to be configured and specified on
 every command.
 
 ## Configuration
-Clin uses configuration using a yaml file located either in the command's
+Clin uses configuration from YAML file located either in the command's
```
Comment (dstockhammer) [inline docs/README.md:64]: from `a` YAML file?

File: docs/README.md [hunk 13]
```diff
@@ -108,9 +106,9 @@ when applying it in bash, the current user will be the admin of the event type
 or subscription.
 
 
-In case of batch processing (see [Batch processing](#batch-processing)), context
+In the case of batch processing (see [Batch processing](#batch-processing)), the context
```
Comment (dstockhammer) [inline docs/README.md:109]: I'd say there the article is superfluous; ie `in case of batch processing`...

File: docs/README.md [hunk 14]
```diff
@@ -160,11 +158,11 @@ process:
 #...
 ```
 
-If a variable, found in manifest is not defined in the context – the process
+If a variable found in the manifest is not defined in the context – the process
```
Comment (dstockhammer) [inline docs/README.md:161]: comma instead of hyphen?

File: docs/README.md [hunk 15]
```diff
@@ -180,7 +178,7 @@ can contain includes of some parts from other files. Cyclic references are not
 permitted
 
 First includes resolved and then variables are processed, which means:
-- included files can contain variables which will be processed from same context
+- included files can contain variables which will be processed from the same context
 - include reference can NOT contain variable, following is invalid:
```
Comment (dstockhammer) [inline docs/README.md:182]: -> `an` include reference can NOT contain `any` variable`s`. The following is invalid:

File: docs/README.md [hunk 16]
```diff
@@ -180,7 +178,7 @@ can contain includes of some parts from other files. Cyclic references are not
 permitted
```
Comment (dstockhammer) [inline docs/README.md:178]: period at end of sentence missing

File: docs/README.md [hunk 17]
```diff
@@ -180,7 +178,7 @@ can contain includes of some parts from other files. Cyclic references are not
...
 
 First includes resolved and then variables are processed, which means:
```
Comment (dstockhammer) [inline docs/README.md:180]: `are` resolved

**Comment by dstockhammer** [comment]
:+1:

**Comment by dstockhammer** [comment]
👍
